### ISSUE TITLE
Darktide

### ISSUE THREAD

**Issue by shoompaps** [Nukem9/dlssg-to-fsr3#157]
Hello I tried this with Darktide. I understand it's a multiplayer game but just went with it to try the mod out. It works really well but the UI elements when moving has severe ghosting or is blurring so much. All the other games I've tried like Cyberpunk, Witcher 3, Spiderman doesn't have this issue. Any thoughts? Thanks

**Comment by ZZZXXX0110** [comment]
Yes Darktide is a coop horde fighting game where there is already a pretty big modding scene existing so I think using this is justified even though its technically a multiplayer game.

I also tried it in Darktide and got the exact same result as described here. Additionally the way the UI artifacts look like resembles the type of artifacts DLSS FG had in its early days, and looks as if FSR3 FG is also applying the frame rate interpolation to all the UI elements when it shouldn't interpolate the UI elements. This is a strong contrast to because most of the 2D spirits in the game that's not UI elements, like smoke and trailing smoke lines behind bullets don't have noticeable artifacts at all and are working as intended.
